### PR TITLE
Fix this.model.selectedItemFor is not a function

### PR DESCRIPTION
**Pull request by viddo**
This fixes the `this.model.selectedItemFor is not a function` that was standing out in trackjs, luckily the error is thrown in an event callback and do not affect that step, but it pollutes the error reports so better fix it.

ping @javierarce for code review

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5366/))

**Comment by javierarce** [comment]
:+1:

**Comment by viddo** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5375/))

File: lib/assets/test/spec/cartodb/common/dialogs/merge_datasets/column_merge/footer_info_view.spec.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      });
+      this.model.get('rightColumns').add({ name: 'RIGHT' });
+      this.view = new FooterInfoView({
+        model: this.model
+      });
+      this.view.render();
+    });
+
+    it('should not have leaks', function() {
+      expect(this.view).toHaveNoLeaks();
```
Comment (javisantana) [inline lib/assets/test/spec/cartodb/common/dialogs/merge_datasets/column_merge/footer_info_view.spec.js]: :__)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5410/))

**Pull request merged**